src/Microsoft.Health.Fhir.Shared.Client/FhirClient.cs

```diff
@@ -291,6 +291,19 @@ public async Task<FhirResponse<Bundle>> SearchPostAsync(string resourceType, Can
             return await CreateResponseAsync<Bundle>(response);
         }
 
+        public async Task<FhirResponse<Resource>> PostAsync(string resourceType, string body, CancellationToken cancellationToken = default)
```

Comment:
>PostAsync [](start = 50, length = 9)

```
var fhirSource = Samples.GetJson("Profile-Patient-uscore");
var parser = new FhirJsonParser();
var patient = parser.Parse<Resource>(fhirSource).ToTypedElement().ToResourceElement();
Patient createdResource = await _client.CreateAsync(patient.ToPoco<Patient>());
```
I went this route to post resources to server which are in string format, gives me some guarantee what I don't mess up json file and it's a valid resource before i post it. On downside you have to do double serialization/deserialization

Reply:
I did start with the _client.CreateAsync.  In this case, because I am testing with some invalid resources, I cannot successfully parse them into a resource object before I POST them.  I need to POST as raw JSON.